blog/_posts/2021-01-19-scala-native-0.4-release.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: blog-detail
+post-type: blog
+by: Wojciech Mazur, Scala Center
+title: A sneak peek into the Scala Native 0.4 release
+---
+
+If you ever thought about running your perfect Scala code in an environment other than JVM, you've probably already met
+Scala Native---a project with a promise of instant startup, low memory usage, interop with C libraries, and competitive performance
+thanks to [Interflow optimizer](https://scala-native.readthedocs.io/en/latest/blog/interflow.html).
+
+However, since there have been no release in over a year, you may ask yourself---is this project even alive?<br>
+With the long-awaited release 0.4.0 it proves it is! 
+Today we are going to discuss the most impacting new features and changes coming in this new major release.
```

Comment:
s/impacting/impactful/ ("impacting" is ungrammatical). but "impactful" is business-speak, so maybe a different word entirely, something more conversational like "exciting" or "major"?